cmd/limactl/copy.go

```diff
@@ -65,6 +65,48 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		verbose = true
 	}
 
+	cpTool := rsync
+	arg0, err := exec.LookPath(string(cpTool))
+	if err != nil {
+		arg0, err = exec.LookPath(string(cpTool))
+		if err != nil {
+			return err
+		}
+	}
+	logrus.Infof("using copy tool %q", arg0)
+
+	var sshArgs, toolArgs []string
+
+	switch cpTool {
+	case scp:
+		sshArgs, toolArgs, err = useScp(args, verbose, recursive)
+		if err != nil {
+			return err
+		}
+	case rsync:
+		toolArgs, err = useRsync(args, verbose, recursive)
+		if err != nil {
+			return err
+		}
+	default:
+		return fmt.Errorf("invalid copy tool %q", cpTool)
+	}
+
+	sshCmd := exec.Command(arg0, append(sshArgs, toolArgs...)...)
+	sshCmd.Stdin = cmd.InOrStdin()
+	sshCmd.Stdout = cmd.OutOrStdout()
+	sshCmd.Stderr = cmd.ErrOrStderr()
+	logrus.Debugf("executing scp (may take a long time): %+v", sshCmd.Args)
```

Comment:
Need to remove "scp" and just log the command instance:

    logrus.Debugf("executing %v (may take a long time)", sshCmd)